### ISSUE TITLE
no client information at consumer group side

### ISSUE THREAD

**Issue by IronLung7** [dpkp/kafka-python#2199]
Using consumer from kafka-python,
there is no client information at consumer group side:

./kafka-consumer-groups.sh --bootstrap-server xxxxxxxx:9092 --describe  --group  aa
Note: This will not show information about old Zookeeper-based consumers.

Consumer group 'aa' has no active members.

TOPIC                          PARTITION  CURRENT-OFFSET  LOG-END-OFFSET  LAG        CONSUMER-ID                                       HOST                           CLIENT-ID
topic-a              0          928             928             0          -                                                 -                              -

**Comment by IronLung7** [comment]
Using kafka-python==2.0.1,  and the client_id is set when initializing consumer.

**Comment by IronLung7** [comment]
If using `consumer.subscribe(topic_l)`,  Client-id will in consumer group describe.
if using `consumer.assign(TopicPartition)`, Client-id will not in consumer group describe.

**Comment by dpkp** [comment]
Correct, `consumer.subscribe` uses broker apis to create a group and join it. `consumer.assign` does not create or use a group. So there is nothing to describe.
